Projects/Features/StorageFeature/Sources/ViewControllers/NoticeDetailViewController.swift

```diff
@@ -54,30 +56,32 @@ public final class NoticeDetailViewController: UIViewController, ViewControllerF
 
 private extension NoticeDetailViewController {
     func inputBind() {
-        viewModel.input.fetchNoticeDetail.onNext(())
+        input.fetchNoticeDetail.onNext(())
 
         collectionView.rx.itemSelected
-            .withLatestFrom(viewModel.output.dataSource) { ($0, $1) }
-            .map { $0.1[$0.0.section].items[$0.0.item] }
-            .bind(with: self) { owner, model in
-                guard !model.link.isEmpty,
-                      let URL = URL(string: model.link) else { return }
-                owner.present(SFSafariViewController(url: URL), animated: true)
-            }
+            .bind(to: input.didTapImage)
             .disposed(by: disposeBag)
     }
 
     func outputBind() {
-        viewModel.output.dataSource
+        output.dataSource
+            .skip(1)
             .bind(to: collectionView.rx.items(dataSource: createDataSource()))
             .disposed(by: disposeBag)
 
-        viewModel.output.imageSizes
+        output.imageSizes
             .skip(1)
-            .subscribe(onNext: { [weak self] _ in
-                self?.indicator.stopAnimating()
+            .subscribe(onNext: { [indicator] _ in
+                indicator?.stopAnimating()
             })
             .disposed(by: disposeBag)
+
+        output.goSafariScene
+            .bind(with: self) { owner, link in
+                guard let URL = URL(string: link) else { return }
+                owner.present(SFSafariViewController(url: URL), animated: true)
+            }
```

Comment:
```suggestion
        output.goSafariScene
            .compactMap { URL(string: $0)
            .bind(with: self) { owner, url in
                owner.present(SFSafariViewController(url: url), animated: true)
            }
```
compactMap 써도 좋을거같아용